homeassistant/components/spider/sensor.py

```diff
@@ -0,0 +1,117 @@
+"""Support for Spider Powerplugs (energy & power)."""
+from datetime import datetime
+
+from homeassistant.components.sensor import STATE_CLASS_MEASUREMENT, SensorEntity
+from homeassistant.const import (
+    DEVICE_CLASS_ENERGY,
+    DEVICE_CLASS_POWER,
+    ENERGY_KILO_WATT_HOUR,
+    POWER_WATT,
+)
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.util import dt as dt_util
+
+from .const import DOMAIN
+
+
+async def async_setup_entry(hass, config, async_add_entities):
+    """Initialize a Spider Power Plug."""
+    api = hass.data[DOMAIN][config.entry_id]
+    entities = []
+
+    for entity in await hass.async_add_executor_job(api.get_power_plugs):
```

Comment:
We reserve the word entity for `Entity` instances. Please rename the variable to eg `plug`.